src/hotspot/cpu/riscv/c2_MacroAssembler_riscv.hpp

```diff
@@ -239,6 +239,14 @@
                         int opc, BasicType bt, uint vector_length,
                         VectorMask vm = Assembler::unmasked);
 
+  void reduce_mul_integer_v(Register dst, Register src1, VectorRegister src2,
```

Comment:
Or maybe `reduce_mul_integral_v` which will be consistent in naming with friends like `reduce_integral_v`?